src/telemetry/Manager.h

```diff
@@ -250,6 +267,11 @@ class Manager final {
     std::unique_ptr<prometheus::Exposer> prometheus_exposer;
 
     std::string cluster_json;
+
+    std::shared_ptr<ZeekCollectable> zeek_collectable;
+    std::unique_ptr<zeek::detail::Flare> collector_flare;
```

Comment:
No strong opinion, but flare could probably be embedded, too.

Reply:
I left it here mostly because of how easy our `Flare` type is to use.